src/routes/LoginEnterprise.test.tsx

```diff
@@ -75,35 +74,6 @@ describe('routes/LoginEnterprise.tsx', () => {
     expect(validate(values).clientSecret).toBe('Invalid client secret.');
   });
 
-  it('should receive a logged-in enterprise account', () => {
```

Comment:
is there a reason why this test case has been removed?

Reply:
It was a test for page transition when already logged in with enterprise.
Since I removed that restriction, this test case is no longer needed.